packages/webapp/src/members/components/temp-member-card-resign-modal.tsx

```diff
@@ -0,0 +1,70 @@
+import React, { useState } from "react";
```

Comment:
Filename is "temp-"? Is that intentional?

Reply:
Probably also rename component from `MemberCard` to `MemberCardResignModal`

Reply:
and update/remove the data-testid

Reply:
Yeah, this wasn't meant to me checked in. Sorry, guys!

Reply:
Removed.